docs/UserGuide.md

```diff
@@ -561,19 +651,25 @@ Furthermore, certain edits can cause the MyBookshelf to behave in unexpected way
 
 ## Glossary
 
-1. `Library User:` The people that are saved into the contact list of *MyBookshelf*. Sometimes referred to as "borrowers".
+1. **Library User**: The people that are saved into the contact list of *MyBookshelf*. Sometimes referred to as "borrowers".
+
+
+2. **Library Manager**: Community Library Managers (CLM) are the people using the MyBookshelf application. CLMs are responsible for adding, storing, and updating the entire library database via *MyBookshelf*.
+
+
+3. **Contact List**: Refers to the list of library users currently stored in the *MyBookshelf* application. It appears in the left column of the User Interface.
 
 
-2. `Library Manager:` Community Library Managers (CLM) are the people using the MyBookshelf application. CLMs are responsible for adding, storing, and updating the entire library database via *MyBookshelf*.
+4. **Book**: Identified by its `BOOKTITLE`. Appears in both the `Borrowed Book List` and the `Library Book List`.
 
 
-3. `Contact List:` Refers to the list of library users currently stored in the *MyBookshelf* application. It appears in the left column of the User Interface.
+5. **Borrowed Book List**: Refers to the list of `Book`(s) currently borrowed by an individual library user.
```

Comment:
Inconsistent naming.

Some parts of user guide uses `user's book list` instead of `borrowed book list`, while other parts uses `borrowed book list`.

Need discussion for this.